devops/README.md

```diff
@@ -1,17 +1,147 @@
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+wget https://packages.microsoft.com/config/debian/10/prod.list -O /etc/apt/sources.list.d/packages-microsoft-com_prod.list
+apt update
+apt install osconfig
+```
+
+## Debian 11 - insiders-fast (AMD64)
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/trusted.gpg.d/packages-microsoft-com_key.asc
+wget https://packages.microsoft.com/config/debian/11/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+apt install osconfig
+```
+
+## Debian 11 - prod (AMD64)
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+wget https://packages.microsoft.com/config/debian/11/prod.list -O /etc/apt/sources.list.d/packages-microsoft-com_prod.list
+apt update
+apt install osconfig
+```
+
+## Debian 12 - insiders-fast (AMD64)
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/trusted.gpg.d/packages-microsoft-com_key.asc
+wget https://packages.microsoft.com/config/debian/12/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+apt install osconfig
+```
+
+## Red Hat Enterprise Linux 7 - insiders-fast (AMD64)
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-rhel7.4-insiders-fast-prod/config.repo
+yum install osconfig
+```
 
-[![E2E](https://github.com/Azure/azure-osconfig/actions/workflows/e2e.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/e2e.yml)
-[![Module Test](https://github.com/Azure/azure-osconfig/actions/workflows/module-test.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/module-test.yml)
+## Red Hat Enterprise Linux 8 - insiders-fast (AMD64)
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-rhel8.0-insiders-fast-prod/
+yum install osconfig
+```
+
+## Red Hat Enterprise Linux 9 - insiders-fast (AMD64)
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/yumrepos/microsoft-rhel9.0-insiders-fast-prod/config.repo
+yum install osconfig
+```
+
+## Rocky Linux 9 - insiders-fast (AMD64)
+```bash
+yum install yum-utils
+yum-config-manager --add-repo https://packages.microsoft.com/config/rocky/9/insiders-fast.repo
+yum install osconfig
+```
+
+## SUSE Linux Enterprise Server 15 - insiders-fast (AMD64)
+```bash
+zypper install yum-utils
+zypper ar -f https://packages.microsoft.com/yumrepos/microsoft-sles15-insiders-fast-prod/config.repo
+zypper install osconfig
+```
+
+## Ubuntu 20.04 - insiders-fast (AMD64)
+```bash
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+wget https://packages.microsoft.com/config/ubuntu/20.04/insiders-fast.list -O /etc/apt/sources.list.d/packages-microsoft-com_insiders-fast.list
+apt update
+apt install osconfig
```

Comment:
Why apt and not apt-get?

Reply:
No reason other than apt is available and usually more user-friendly